event-based-simulators/main/event_based_simulators.py

```diff
@@ -8,21 +8,36 @@
 def current_timestamp(format = "%Y-%m-%dT%H:%M:%S.%f"):
     return datetime.utcnow().strftime(format)[:-3] + 'Z'
 
-logging.basicConfig(format='%(asctime)s %(name)s:%(message)s', level=logging.INFO)
+cumulocityAPI = CumulocityAPI()
+oeeAPI = OeeAPI()
+
+#TODO: Parameter to delete all Profiles
```

Comment:
that line should be obsolete now, shouldnt it?

Reply:
Done